build/preprocessor.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+/* jshint node: true */
+/* global exports,
```

Comment:
Remove /* global exports, require */ because
/* jshint node: true */ is able to cover node's keyword such as ```exports```, ```require```